### PR TITLE
fix: Improve "add block" text in NUX onboarding

### LINKED ISSUE #7506: Tooltip: Suggests to click on "Add Block", which is not visible on screen
**Describe the bug**
When the tool reads: 
"Welcome to the wonderful world of blocks! **Click on 'Add block'** to insert different kinds of content (...)" 

**To Reproduce**
1. Use a clean install
2. Add a new post
3. wait until Tool tip appears
4. Find the place where there is a "Add Block" mentioned 

**Expected behavior**
The blue dot points to the inserter + so the wording could say: **Click on the + sign** to insert different kinds of content.

Also, possible to have the "Add Block" text show above the inserter to make clear that's where people should click. 

**Screenshots**
screen shot 2018-06-22 at 10 32 14 am

**Desktop (please complete the following information):**
 - OS: X 10.11.6
 - Browser Chrome
 - Version Version 67.0.3396.87 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by tofumatt**
Fixes #7506

## Description
Updated the text to reference the "+" button instead of a non-existent "Add Block" button; also took the time to fix the British quotes and clarify the language. 🇺🇸!!!

## How has this been tested?
Ran locally and updated jest snapshots.

### Before
screen shot 2018-06-22 at 10 32 14 am

### After
screenshot 2018-06-23 17 32 25

## Types of changes
Text changes.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation.

### CODE REVIEW

**Comment by afercia** [comment]
Hm, regarding the new text:
`Click the “+” button (highlighted)`

the point is screen reader users have no clue what a "+" button is in the Gutenberg UI, as the plus is only visual while the accessible name (given in this case by the aria-label) says "Add block". Also, they can't see what is highlighted or not 🙂 

screen shot 2018-06-23 at 18 44 50

The button is announced as "Add block" and should be referenced as such. This was also mentioned in the first discussion around the NUX, see https://github.com/WordPress/gutenberg/issues/3670#issuecomment-347491998
> The tip text should avoid any visual/positional reference; instead, it should try to describe in a simple language what a control is, where it is placed and what it does.

An option could be clarifying in the new wording that the button's name is "Add block" and visually appears as a plus icon. In the most simple way, something like `Click the "Add block" button with the plus icon ...` perhaps? 

Aside: this is one more downside of icon-only controls: when they need to be "explained" to users, that needs to happen referencing both their appearance and their action name.

**Comment by tofumatt** [comment]
> The button is announced as "Add block" and should be referenced as such. This was also mentioned in the first discussion around the NUX, see #3670 (comment)

I thought about that, and what I'd really like is for there to be off-screen text that said "Add block". I'm lost on how to do that with our gettext function. I'll have a look, there must be prior art.

But showing that text to sighted users is decidedly *inaccessible*, because it forces them to hunt for a UI element ("Add block") that doesn't exist. 😄

**Comment by afercia** [comment]
> But showing that text to sighted users is decidedly inaccessible, 

A bit controversial, maybe 🙂the tooltip is displayed on hover and focus, users will learn the name of the "plus" button is "Add block". Ideally, the NUX tips should display the referenced UI controls in their hovered/focused state (alas, not possible for the Preview button I guess).

Regardless, I'd say that hiding important information assuming that is useful only for screen reader users is just that, an assumption. Clear, visible, text benefits everyone. For example, not all screen reader users are 100% blind...

**Comment by jasmussen** [comment]
A similar challeng has popped up with the modal component in development, where focus is automatically set on the Close button. In https://github.com/WordPress/gutenberg/pull/6261#issuecomment-399355614 I suggested setting focus on the modal itself, so the first tabbing would go to the tab button.

What wizardry is being employed here, and could we do the same for the modal?

**Review by karmatosed — DISMISSED** [review]
I am not sure we should say 'highlighted' cc @michelleweber for copy changes here before we merge those. We totally can work on things but I feel adjusting here needs at least an editorial review first from the person that worked on the copy.

**Comment by tofumatt** [comment]
Right on; updated the text a bit to remove the highlighted bit, but I want to make it clear the "+" button should be searched for so I settled on what's posted.

**Comment by michelleweber** [comment]
There's no reason not to split the difference, is there? E.g., 

"Click the "+" button to add a new block. There are blocks available for all kind of content -- insert text, headings, images, lists, and lots more."

**Comment by tofumatt** [comment]
> There's no reason not to split the difference, is there? E.g.,
> 
> "Click the "+" button to add a new block. There are blocks available for all kind of content -- insert text, headings, images, lists, and lots more."

As @afercia pointed out: there is–screen reader users will not see "+" button so we should reference the "Add block" text as well. How about:

```
Welcome to the wonderful world of blocks! Click the “+” (“Add block”) button to add a new block. There are blocks available for all kind of content: you can insert text, headings, images, lists, and lots more!
```

**Comment by michelleweber** [comment]
Right! Thank you. I think your version is right on.

**Comment by tofumatt** [comment]
Editorial-approved copy added–should be good to go!

**Review by noisysocks — APPROVED** [review]
Code LGTM!

**Comment by tofumatt** [comment]
@karmatosed Dismissed your review because editorial and a developer approved it and your concern was with editorial. Let me know if I'm wrong and shouldn't have done that, but if dismissing it was okay then enjoy one less PR to review 😉

**Comment by karmatosed** [comment]
@tofumatt all good just wanted to get the copy review insight here, didn't need me :) Thanks.

**Comment by karmatosed** [comment]
Noting PR for this here: https://github.com/WordPress/gutenberg/pull/7588

**Pull request merged**
